landlab/components/groundwater/dupuit_percolator.py

```diff
@@ -0,0 +1,197 @@
+# -*- coding: utf-8 -*-
+"""
+GroundwaterDupuitPercolator Component
+
+@author: G Tucker
+"""
+
+import numpy as np
+from landlab import Component
+from landlab.utils import return_array_at_node, return_array_at_link
+from landlab.grid.mappers import map_mean_of_link_nodes_to_link
+
+
+ACTIVE_LINK = 0
+
+
+class GroundwaterDupuitPercolator(Component):
+    """
+    Simulate groundwater flow in a shallow unconfined aquifer.
+
+    The GroundwaterDupuitPercolator uses the Dupuit approximation that the
+    hydraulic gradient is equal to the slope of the water table.
+
+    Parameters
+    ----------
+    grid: ModelGrid
+            Landlab ModelGrid object
+    hydraulic_conductivity: float, field name, or array of float
+            saturated hydraulic conductivity, m/s
+            Default = 0.01 m/s
+    recharge_rate: float, field name, or array of float
+            Rate of recharge, m/s
+            Default = 1.0e-8 m/s
+
+    Examples
+    --------
+    >>> from landlab import RasterModelGrid
+    >>> mg = RasterModelGrid((3, 3))
+    >>> z = mg.add_zeros('node', 'topographic__elevation')
+    >>> gdp = GroundwaterDupuitPercolator(mg)
```

Comment:
I think you need to show the component import from `landlab.components`. Not that this won't work at the moment without new `__init__.py` files being created/existing ones modified.